src/Scene/NodeProcess.js

```diff
@@ -78,47 +72,36 @@ NodeProcess.prototype.checkNodeSSE = function checkNodeSSE(node) {
 
 NodeProcess.prototype.subdivideNode = function subdivideNode(node, camera, params) {
     if (!node.pendingSubdivision && node.noChild()) {
-        var bboxes = params.tree.subdivideNode(node);
+        const bboxes = params.tree.subdivideNode(node);
         node.pendingSubdivision = true;
 
         for (var i = 0; i < bboxes.length; i++) {
-            var args = {
+            const quadtree = params.tree;
+            const command = {
+                /* mandatory */
+                requester: node,
                 layer: params.layersConfig.getGeometryLayers()[0],
+                priority: 10000,
+                /* specific params */
                 bbox: bboxes[i],
+                type: quadtree.type,
             };
-            var quadtree = params.tree;
 
-            quadtree.interCommand.request(args, node).then((child) => {
-                var colorTextureCount = 0;
-                var paramMaterial = [];
-                var layer;
-                var j;
-
-                child.matrixSet = [];
+            quadtree.scheduler.execute(command).then((child) => {
+                let colorTextureCount = 0;
+                const paramMaterial = [];
 
                 // update wmts
-                var colorLayers = params.layersConfig.getColorLayers();
+                const colorLayers = params.layersConfig.getColorLayers();
 
                 // update Imagery wmts
-                for (j = 0; j < colorLayers.length; j++) {
-                    layer = colorLayers[j];
-                    const tileMatrixSet = layer.options.tileMatrixSet;
-
-                    if (tileMatrixSet && !child.matrixSet[tileMatrixSet]) {
-                        child.matrixSet[tileMatrixSet] = this.projection.getCoordWMTS_WGS84(child.tileCoord, child.bbox, tileMatrixSet);
-                    }
-
+                for (const layer of colorLayers) {
```

Comment:
`for (const layer of colorLayers)`
Warning : problem with this notation with other framework (ember.js)